src/Net/HostnameToIpResolver.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RapidCore.Net
```

Comment:
I chose .Net since this is what it's called in dotnet core as well - renamed to Network as it does indeed sound better.